### PR TITLE
Ice added.

### PR DESCRIPTION
**Pull request by acmgit**
Hello, i've added Ice for the summer.

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -117,6 +118,727 @@ food.module("lemon",function()
 	})
 end, true)
 
+-- Ice
+food.module("ice-bowl", function()
```
Comment (rubenwardy) [inline init.lua]: ice_bowl

File: init.lua [hunk 2]
```diff
@@ -117,6 +118,727 @@ food.module("lemon",function()
...
+	minetest.register_craftitem(":food:icebowl",{
```
Comment (rubenwardy) [inline init.lua]: food_sweet:ice_bowl

item names should be consistent with module names

File: init.lua [hunk 3]
```diff
@@ -117,6 +118,727 @@ food.module("lemon",function()
...
+		description = S("Ice-Bowl"),
+		inventory_image = "food_sweet_ice_bowl.png",
+		groups = {food_bowl=1, food_ice=1}
```
Comment (rubenwardy) [inline init.lua]: also food_ice_bowl=1
unless it's possible to have AND with groups in craft recipes?

File: init.lua [hunk 4]
```diff
@@ -117,6 +118,727 @@ food.module("lemon",function()
...
+		description = S("Icewafer Blackberry"),
+		inventory_image = "food_sweet_ice_wafer_blackberry.png",
+		on_use = food.item_eat(2),
+		groups = {food_ice = 1}
+	})
+	food.craft({
+		output = "food_sweet:ice_wafer_blackberry",
+		recipe = {
+			{"food_sweet:ice_ball_blackberry"},
+			{"food:wafer"}
```
Comment (rubenwardy) [inline init.lua]: group:food_wafer

**Review by rubenwardy — COMMENTED** [review]
Also, you should remove the `false` off all of the module calls here

**Comment by rubenwardy** [comment]
Thanks for the contribution though! Sorry it took so long :/

File: init.lua (hunk 3 above)
Comment (acmgit) [reply init.lua]: The new Bowl is an alternative to the existing Bowl. The Recipes for the Cups but should take any  of this Bowls ;-).

**Comment by acmgit** [comment]
Hello,
thx for your comments. I've fixed it.
Nevermind about the long time, now we communicate ;-).

**Pull request closed without merging**